src/persist-client/src/cli/inspect.rs

```diff
@@ -778,8 +778,14 @@ struct T;
 #[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Ord)]
 struct D(i64);
 
-pub(crate) static KVTD_CODECS: Mutex<(String, String, String, String)> =
-    Mutex::new((String::new(), String::new(), String::new(), String::new()));
+pub(crate) static KVTD_CODECS: Mutex<(String, String, String, String, Option<&'static str>)> =
```

Comment:
nit: feels like this deserves a newtype or some simple struct to explain the semantics of the last element a bit more

Reply:
done!